README.md

```diff
@@ -1065,6 +1065,20 @@ will be rendered as
 
 (some unimportant HTML attributes have been removed for simplicity)
 
+### Turbo submits with
+There is a turbo attribute [(data-turbo-submits-with)](https://turbo.hotwired.dev/reference/attributes) that replaces the content of a submit button while the request is processing.
+```erb
+<%= f.button "Save", submits_with: :spinner %>
+```
+
+```html
+<button class="btn btn-secondary" data-turbo-submits-with="<div class="spinner-border d-block mx-auto"; role="status" style="--bs-spinner-width: 1lh; --bs-spinner-height: 1lh;"></div>" name="button" type="submit">Save</button>
+```
+
+Use `submits_with: :spinner` to render a default bootstrap spinner or pass your own HTML.
+This only works on `f.button` or `f.primary` not on `f.submit` and forces `render_as_button: true` on `f.primary`.
```

Comment:
I would move this line up to near the top of this section, because some of us don't read the whole section before we start coding. ;-)